agent/mibgroup/Rmon/rows.c

```diff
@@ -67,7 +67,7 @@ rowapi_delete(RMON_ENTRY_T * eold)
     }
 
     /*
-     * delete it in users's sence 
+     * delete it in users's sense
```

Comment:
Although I'm not a native speaker: this doesn't look right to me.

Reply:
sense is probably correct. It should be user's though.

Reply:
Added as an additional commit.